### ISSUE TITLE
How to add a new version of a package to the store?

### ISSUE THREAD

**Issue by lorenzo** [dhall-lang/dhall-lang#1155]
I recently released `dhall-nomad` `v1.0.1`, but I'm not clear how to publish the new version to the store.

Are there any instructions available?

Thanks!

**Comment by Gabriel439** [comment]
Yes, you can find the instructions here: https://github.com/dhall-lang/dhall-lang/blob/master/nixops/README.md#adding-a-package-to-storedhall-langorg

**Comment by lorenzo** [comment]
I tried that earlier but I get this error:

```bash
nixops> nix-env --install --attr haskellPackages.dhall-nixpkgs
error: attribute 'haskellPackages' in selection path 'haskellPackages.dhall-nixpkgs' not found
```

I suspect that instruction is out of date or maybe it is missing some other context that I don't know

**Comment by Gabriel439** [comment]
@lorenzo: You might need a newer version of `nixpkgs`, specifically release 20.09 or newer
